### PR TITLE
Cndit 227 validation

### PR DESCRIPTION
**Pull request by ndduc01**
- Validation pipe line
 - Configuration for Mssql and kafka 
 - Contain HL7v2 validator
 - Simple implementation of CSV validator

### CODE REVIEW

File: report-service/src/main/java/gov/cdc/dataingestion/validation/controller/ValidationController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    String topicName;
+
+    @Autowired
+    private RawELRRepository rawELRRepository;
+    KafkaProducerService kafkaProducerService;
+    public ValidationController(KafkaProducerService kafkaProducerService) {
+        this.kafkaProducerService = kafkaProducerService;
+    }
+    @RequestMapping(value = "/elr-simulator", method = RequestMethod.POST)
+    public ResponseEntity<?> saveELRMessage(@RequestBody String payload) {
```
Comment (ndduc01) [inline report-service/src/main/java/gov/cdc/dataingestion/validation/controller/ValidationController.java:31]: This is just for testing, will remove it once we can test the entire pipeline

**Review by sdeusch — COMMENTED** [review]
Also add files in .idea/ to .gitignore

File: report-service/sql-script/elr_raw.sql
Comment (sdeusch) [inline report-service/sql-script/elr_raw.sql:1]: _primary key_ is always _not null_ , you can drop the _not_null_

Comment (sdeusch) [inline report-service/sql-script/elr_raw.sql:1]: created_on datetime not null default getdate()

File: report-service/sql-script/elr_validated.sql
Comment (sdeusch) [inline report-service/sql-script/elr_validated.sql:1]: same as above

File: report-service/src/main/java/gov/cdc/dataingestion/validation/model/ValidatedELRModel.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    private String rawMessage;
+
+    @Column(name = "message_type")
+    private String messageType;
+
+    @Column(name = "message_version")
+    private String messageVersion;
+
+    @Transient
+    private Timestamp created_on;
```
Comment (bspuli) [inline report-service/src/main/java/gov/cdc/dataingestion/validation/model/ValidatedELRModel.java:30]: please update to camel case

**Review by bspuli — APPROVED** [review]
LGTM

**Pull request merged**